doc/tracepath.xml

```diff
@@ -56,21 +56,21 @@ xml:id="man.tracepath">
     <info>
       <title>DESCRIPTION</title>
     </info>
-    <para>It traces path to
+    <para>It traces the networkpath to
```

Comment:
networkpath is not a word - needs space

Reply:
Thank you for spotting this. I added a space and pushed again.